apps/blazor-test-app/Properties/launchSettings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "profiles": {
+    "Microsoft Teams (browser)": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "launchUrl": "https://teams.microsoft.com/l/app/0794bd56-95ae-4835-9588-d81b26ec3d8d?installAppPackage=true&webjoin=true&appTenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&login_hint=noahdarveau@microsoft.com",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "dotnetRunMessages": "true",
+      "applicationUrl": "https://localhost:44302;http://localhost:2544",
+      "hotReloadProfile": "aspnetcore"
+    },
+    "WSL": {
+      "commandName": "WSL2",
+      "launchBrowser": true,
+      "launchUrl": "https://teams.microsoft.com/l/app/0794bd56-95ae-4835-9588-d81b26ec3d8d?installAppPackage=true&webjoin=true&appTenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&login_hint=noahdarveau@microsoft.com",
```

Comment:
should your alias be in the login hint?

Reply:
(in both launchUrls)

Reply:
Who put that there?! I sure didn't. Anyways, fixed!